products/compute/ansible.yaml

```diff
@@ -43,6 +45,8 @@ datasources: !ruby/object:Overrides::ResourceOverrides
     exclude: true
   RegionDiskType: !ruby/object:Overrides::Ansible::ResourceOverride
     exclude: true
+  ResourcePolicy: !ruby/object:Overrides::Ansible::ResourceOverride
+    exclude: true
```

Comment:
Can you exclude these resources in the `overrides` list below as well? Sorry, I forgot they were distinct lists.